test-data/unit/fixtures/callable.pyi

```diff
@@ -0,0 +1,24 @@
+from typing import builtinclass, Tuple, TypeVar, Generic, Union
```

Comment:
@builtinclass is no longer a thing, see #599. Just remove this import, you're not using it anyway.